menas/ui/components/dataset/datasetDetail.controller.js

```diff
@@ -39,6 +40,18 @@ sap.ui.define([
       let cont = new ConformanceRuleDialog(this);
       let view = this.getView();
 
+      view.byId("monitoring-multiheader-info").setHeaderSpan([2,1]);
```

Comment:
Can these configs be specified through the view?

Reply:
Unfortunately not